test/clj/harja/testi.clj

```diff
@@ -310,17 +310,45 @@
                    \"urakka-id\" = (SELECT id FROM urakka WHERE nimi = 'Helsingin väyläyksikön väylänhoito ja -käyttö, Itäinen SL')
                    AND id NOT IN (SELECT \"toimenpide-id\" FROM vv_hinnoittelu_toimenpide) LIMIT 1;"))))
 
+(defn hae-helsingin-reimari-toimenpiteet-molemmilla-hinnoitteluilla [& options]
```

Comment:
Mielestäni options ei tarvitse olle rest-parametri, vaan toimisi yksinkertaisemmin ihan yksittäisenä parametrina (optiot), jonka oletetaan olevan mappi. Tällöin siitä voidaan myös destruktoida suoraan "limit" ja "hintaryhma". Helpottaa hahmottamaan heti funktion määrittelyssä, mitä optioita hyväksytään.